recipes/openllm/meta.yaml

```diff
@@ -0,0 +1,81 @@
+{% set name = "openllm" %}
+{% set version = "0.4.31" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/openllm-{{ version }}.tar.gz
+  sha256: cd2850a6c46b8f801325265a8b7e91ee193a8acfb848deccb7efc6273ee91895
+
+build:
+  skip: true  # [win or osx]
+  skip: true  # [py<38]
+  entry_points:
+    - openllm = openllm_cli.entrypoint:cli
+    - openllm-dive-bentos = openllm_cli.extension.dive_bentos:cli
+    - openllm-get-containerfile = openllm_cli.extension.get_containerfile:cli
+    - openllm-get-prompt = openllm_cli.extension.get_prompt:cli
+    - openllm-list-bentos = openllm_cli.extension.list_bentos:cli
+    - openllm-list-models = openllm_cli.extension.list_models:cli
+    - openllm-playground = openllm_cli.extension.playground:cli
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python
+    - hatchling ==1.18.0
+    - hatch-vcs ==0.3.0
+    - hatch-fancy-pypi-readme ==23.1.0
+    - pip
+  run:
+    - python
+    - bentoml >=1.1.10
+    - transformers >=4.35.0
+    - openllm-client >=0.4.31
+    - openllm-core >=0.4.31
+    - safetensors
+    - optimum >=1.12.0
+    - accelerate
+    - ghapi
+    - einops
+    - sentencepiece
+    - scipy
+    - python-build <1
+    - click >=8.1.3
+    - cuda-python
+    - bitsandbytes <0.42
+# Required for openllm-playground --help
+# https://github.com/bentoml/OpenLLM/blob/main/openllm-python/pyproject.toml#L119
+    - jupyter
```

Comment:
This should be a different output then. A project that is meant for production shouldn't have debug dependencies.

Reply:
@xhochy - 
Makes sense - given that playground had a client entry point and help command as part of the main package - I wasn't sure how to treat it, as a feature or a debug / separate from production. 
    - openllm-playground = openllm_cli.extension.playground:cli
All the same - let me rework this PR

Thanks for your time, I will revert back.